### PR TITLE
Rename versionInfo to version_info

### LINKED ISSUE #64: Why camel case for version info
@gautam-kotian-sociomantic

> The AA containing the version information (https://github.com/sociomantic-tsunami/makd/blob/v1.x.x/Version.d.tpl#L35) is named with camelCase. This seems weird because we generally use snake_case for variables.

### PR DESCRIPTION
**Pull request by gkotian**
Also deprecate versionInfo in the process.

Fixes #64.

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
I would do this for v1.x.x and completely remove `versionInfo` in v2.x.x instead.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 ======================
 
-* The `Version` alias to `versionInfo` provided by the `Version` module has been removed.
-  All remaining usage can be replaced with `versionInfo`.
+* The AA containing all the automatically deduced version information was
+  earlier called `versionInfo` (provided by the `Version` module). This name is
+  now deprecated. All usage of this name should be replaced with `version_info`.
+
+* The `Version` alias to the now deprecated `versionInfo` has been removed. All
+  remaining usage should be replaced with `version_info`.
```
Comment (leandro-lucarella-sociomantic) [inline RELEASE_NOTES.md]: Normally deprecations go into their own section. But I guess this will have to be changed when rebased to v1.x.x anyway.

**Comment by leandro-lucarella-sociomantic** [comment]
I based the PR in v1.x.x, please rebase. After this we can merge to v2.x.x and remove the deprecated symbol.

**Review by leandro-lucarella-sociomantic — CHANGES_REQUESTED** [review]
Please rebase.

**Comment by gkotian** [comment]
Rebased.

File: Version.d.tpl [hunk 2]
```diff
@@ -32,16 +32,20 @@ else
     alias char[] _tpl_istring;
 }
 
-_tpl_istring[_tpl_istring] versionInfo;
-deprecated("Please use @MODULE@.versionInfo instead (part of the D2 transition)")
-alias versionInfo Version;
+_tpl_istring[_tpl_istring] version_info;
+
+deprecated("'versionInfo' is deprecated. Please rename it to 'version_info'")
```
Comment (llucax) [inline Version.d.tpl]: Why not using the `@MODULE@` in this message too?

Comment (leandro-lucarella-sociomantic) [reply Version.d.tpl]: ping

Comment (gkotian) [reply Version.d.tpl]: No reason. Will add it.

**Comment by gkotian** [comment]
Updated with `@MODULE@` in the deprecation message.

**Review by leandro-lucarella-sociomantic — APPROVED** [review]

**Pull request merged**